doc/computing/parallelism.rst

```diff
@@ -146,33 +156,23 @@ Note that:
   only use ``<LIB>_NUM_THREADS``. Joblib exposes a context manager for
   finer control over the number of threads in its workers (see joblib docs
   linked below).
-- Joblib is currently unable to avoid oversubscription in a
-  multi-threading context. It can only do so with the ``loky`` backend
-  (which spawns processes).
+- `threadpoolctl` internally manages the numbers of threads used by OpenMP
+  and BLAS implementations for scikit-learn implementations.
 
 You will find additional details about joblib mitigation of oversubscription
 in `joblib documentation
 <https://joblib.readthedocs.io/en/latest/parallel.html#avoiding-over-subscription-of-cpu-resources>`_.
 
+You will find additional details about parallelism in numerical python libraries
+in `this document from Thomas J. Fan <https://thomasjpfan.github.io/parallelism-python-libraries-design/>`_.
 
 Configuration switches
 -----------------------
 
 Python runtime
 ..............
 
-:func:`sklearn.set_config` controls the following behaviors:
-
-`assume_finite`
-~~~~~~~~~~~~~~~
-
-Used to skip validation, which enables faster computations but may lead to
-segmentation faults if the data contains NaNs.
-
-`working_memory`
-~~~~~~~~~~~~~~~~
-
-The optimal size of temporary arrays used by some algorithms.
+:func:`sklearn.set_config` controls the following behaviors.
```

Comment:
Here, I think it makes sense to explain how `sklearn.{set_config,get_config,config_context}` can be used and point to their documentation rather than duplicating parameters' documentation.

What do you think?

Reply:
Make sense. We might only add which behaviours can be controlled: finiteness, memory, type of output, etc.

Reply:
ede91d210b1fd7bfd0d388069297e3f62c6ed2ea gave some precisions.